docs/common-bugs.rst

```diff
@@ -187,6 +187,14 @@ Failing hardware
 :Solution: It looks like a hardware failure. Please, check your hardware.
 :Example: `rhbz#1685047 <https://bugzilla.redhat.com/show_bug.cgi?id=1685047>`_
 
+LVM on disks with inconsistent sector size
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: Error with message mentioning "inconsistent sector size".
+:Solution: LVM is now demanding that all disks have consistent sector size, otherwise they can't
+    be used together. Please look on
+    `rhbz#1669751 <https://bugzilla.redhat.com/show_bug.cgi?id=1669751>`_ for more info.
```

Comment:
The link doesn't provide useful recommendations. It is mentioned in the example below, I think that is enough. Instead of the link, I would recommend to change the disk selection or the sector sizes of selected disks.

Reply:
It does not but it's the bug where they decided to go this way. I would like to have it here but I'll try to change the wording.